src/app/patient/patient.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "../person/person.hpp"
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+enum class Allergent
+{
+    Allergent = 0,
+    AnotherAllergent = 1,
+    SomeAllergent = 2,
+    DifferentAllergent = 3
+};
+
+class Patient : public Person
+{
+    Patient() = delete;
+    Patient(const std::string name, const std::string last_name, const std::string pesel,
+            const std::unordered_set<Allergent> allergents);
```

Comment:
Allergens should be optional when initialising object.

Reply:
applied